guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -150,7 +150,7 @@ endif::[]
 # systemctl stop rsyslog
 # systemctl stop auditd
 # package-cleanup --oldkernels --count=1
-# {package-clean} all
+# {project-package-clean} all
```

Comment:
Is this supposed to be `client-*`, ie. same as above?

Reply:
Changed to "dnf clean" because docs were mixing "project" and "client" attributes before. the procedure is meant for EL8.